aggregator/src/helpers/networkConfig.ts

```diff
@@ -0,0 +1,9 @@
+// TODO (merge-ok) Replace this with client package import
+import { NetworkConfig, getConfig } from "../../../clients/src/NetworkConfig.ts";
```

Comment:
This and similar TODOs can be done once a new version of the client package is published.

We can also leave this as is and do local linking do keep things simple.

Reply:
Moved these to `bls-wallet-clients` imports, can just use `yarn link` in other spots. Unsure what to do for deno, may need to `-alpha` tag npm releases to test out?